### PR TITLE
Isync 1.3.0

### PR DESCRIPTION
**Pull request by ayatakesi**
fix termux_step_pre_configure in build.sh

### CODE REVIEW

File: packages/isync/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://isync.sourceforge.net
+TERMUX_PKG_DESCRIPTION="IMAP and MailDir mailbox synchronizer"
+TERMUX_PKG_VERSION=1.3.0
+TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/project/isync/isync/${TERMUX_PKG_VERSION}/isync-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=8d5f583976e3119705bdba27fa4fc962e807ff5996f24f354957178ffa697c9c
+TERMUX_PKG_DEPENDS="readline-dev"
+TERMUX_PKG_RECOMMENDS="openssl-dev, libsasl-dev, libdb-dev, libzopfli"
```
[the remark this answers is not in the document]
Comment (ayatakesi) [reply packages/isync/build.sh]: I set TERMUX_PKG_EXTRA_CONFIGURE_ARGS to "--without-sasl ac_cv_header_db=no".

File: packages/isync/isync.patch [hunk 2]
```diff
@@ -0,0 +1,83 @@
+diff -u -r isync-1.3.0.orig/src/compat/config.c isync-1.3.0/src/compat/config.c
+--- isync-1.3.0.orig/src/compat/config.c	2017-10-01 17:42:35.000000000 +0900
++++ isync-1.3.0/src/compat/config.c	2018-02-05 14:08:09.779837587 +0900
```
[the remark this answers is not in the document]
Comment (ayatakesi) [reply packages/isync/isync.patch]: I had corrected this, hopefully...

File: packages/isync/isync.patch [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+@@ -451,7 +451,7 @@
+ 					goto gotstor;
+ 			box->local_store_path = my_strndup( path, pl );
+ 			/* derive a suitable name */
+-			if (!strcmp( box->local_store_path, "/var/mail/" ) || !strcmp( box->local_store_path, "/var/spool/mail/" )) {
++			if (!strcmp( box->local_store_path, "/data/data/com.termux/files/usr/var/mail/" ) || !strcmp( box->local_store_path, "/data/data/com.termux/files/usr/var/spool/mail/" )) {
```
[the remark this answers is not in the document]
Comment (ayatakesi) [reply packages/isync/isync.patch]: done.

File: packages/isync/isync.patch [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
++  return password;
++}
++#endif
++
+ static imap_cmd_t *
+ new_imap_cmd( int size )
+ {
+diff -u -r isync-1.3.0.orig/src/socket.c isync-1.3.0/src/socket.c
+--- isync-1.3.0.orig/src/socket.c	2017-10-01 17:42:35.000000000 +0900
++++ isync-1.3.0/src/socket.c	2018-02-05 14:08:37.529837576 +0900
```
[the remark this answers is not in the document]
Comment (ayatakesi) [reply packages/isync/isync.patch]: done.

File: packages/isync/isync.patch [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+@@ -398,7 +398,7 @@
+ 				_exit( 127 );
+ 			close( a[0] );
+ 			close( a[1] );
+-			execl( "/bin/sh", "sh", "-c", conf->tunnel, (char *)0 );
++			execl( "/data/data/com.termux/files/usr/bin/sh", "sh", "-c", conf->tunnel, (char *)0 );
```
[the remark this answers is not in the document]
Comment (ayatakesi) [reply packages/isync/isync.patch]: done.

**Comment by fornwall** [comment]
Thanks! The package is now available for installation.

**Pull request merged**